liquibase-core/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -45,6 +44,12 @@ public class ChangeSet implements Conditional, ChangeLogChild {
      */
     private CheckSum storedCheckSum;
 
+    public static final String AND = " AND ";
```

Comment:
I think these should be private, since they're just used in here and any global use of those sort of strings should belong somethwere other than ChangeSet

Reply:
Got it